container-core/src/main/java/com/yahoo/container/jdisc/ThreadedRequestHandler.java

```diff
@@ -97,9 +97,12 @@ protected ThreadedRequestHandler(Executor executor, Metric metric, boolean allow
         dimensions.put("port", Integer.toString(uri.getPort()));
         String handlerClassName = getClass().getName();
         dimensions.put("handler-name", handlerClassName);
+        dimensions.putAll(extraDimensions);
         return this.metric.createContext(dimensions);
     }
 
+    private Metric.Context contextFor(Request request) { return contextFor(request, Map.of()); }
```

Comment:
What is the difference between Map.of() and Collections.emptyMap()?

Reply:
`Map.of()` was introduced in Java 9. The returned `Map` instance from both have the same semantics (immutable etc). I prefer the new syntax as it is shorter.

Reply:
When you static import the `emptyMap()` function, I think that is more describing and almost as short. But this is just nit of course. :)

Reply:
Agree - if static import is used `emptyMap()` is more readable 👍